### ISSUE TITLE
currentPageReportTemplate not narrating while using a screen reader

### ISSUE THREAD

**Issue by tj-boyle** [primefaces/primeng#9750]
I'm submitting a ... (check one with "x")

[x] bug report => Search github for a similar issue or PR before submitting
[] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35

Current behavior
Rendered currentPageReportTemplate does not have an aria-label, so it does not work with Screen Readers.

Expected behavior
paginator current page report template has aria-label, and narrates contents with a Screen Reader
